README.md

```diff
@@ -1,59 +1,67 @@
-# Substrate Connect
 
-Substrate Connect offers an innovative way to interact with 
-[Substrate](https://substrate.dev/)-based blockchains directly in your browser, 
-eliminating the need for an RPC server. By leveraging the 
-[smoldot](https://github.com/smol-dot/smoldot) WASM light client, it ensures a 
-secure and efficient connection to the blockchain network without dependency 
-on specific third parties.
+<br /><br />
```

Comment:
lets put a beautiful banner at the top too :sunglasses: 

![image](https://github.com/user-attachments/assets/e1f3b52a-dc4e-40f4-8631-57a3244095af)